### ISSUE TITLE
DIND upgrade from 18.03 to 19.03

### ISSUE THREAD

**Issue by frostyandy2k** [llunaplanet/12factor-dojo#7]
Seems it requires python 3, otherwise:
 ```
> [3/5] RUN apk update &&     apk add bash git make gcc py-pip python-dev libc-dev libffi-dev openssl-dev &&     pip install docker-compose:                 
#6 0.210 fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz                                                                         
#6 1.298 fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz                                                                    
#6 1.737 v3.12.0-216-gaf4199bc57 [http://dl-cdn.alpinelinux.org/alpine/v3.12/main]                                                                            
#6 1.737 v3.12.0-221-gbea860da8e [http://dl-cdn.alpinelinux.org/alpine/v3.12/community]
#6 1.737 OK: 12750 distinct packages available
#6 1.887 ERROR: unsatisfiable constraints:
#6 1.887   python-dev (missing):
#6 1.902     required by: world[python-dev]
```

**Comment by frostyandy2k** [comment]
May work with python2-dev or python3-dev

**Comment by llunaplanet** [comment]
I merged the PR, it's now working fine with 19.03 and python3 #6
